Frameworks/CoreFoundationAdditions/_NSCFTemporaryRootObject.mm

```diff
@@ -16,4 +16,8 @@
 #include "_NSCFTemporaryRootObject.h"
 
 @implementation _NSCFTemporaryRootObject
+- (void)doesNotRecognizeSelector:(SEL)selector {
+    // According to the reference platform documentation, this method must never return.
+    abort();
```

Comment:
std::runtime_error would be a resonable alternative.

---
In reply to: [71576020](https://github.com/Microsoft/WinObjC/pull/671#discussion_r71576020) [](ancestors = 71576020,71575314,71574896,71574472)